### ISSUE TITLE
NullReferenceException occurs when adding a VisualElement after calling VisualElements.Clear()

### ISSUE THREAD

**Issue by carlmichaelbenson** [beto-rodriguez/Live-Charts#356]
#### How to reproduce?

To see this issue, run the following xaml and codebehind...

MainWindow.xaml

```
<Window x:Class="LiveCharts_nugetUpdater.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:LiveCharts_nugetUpdater"
        xmlns:lc="clr-namespace:LiveCharts.Wpf;assembly=LiveCharts.Wpf"
        mc:Ignorable="d"
        Title="MainWindow" Height="350" Width="525" Loaded="Window_Loaded">
    <Grid>
        <lc:CartesianChart Name="TheChart">
            <lc:CartesianChart.Series>
                <lc:LineSeries Values="1,2,3,5" />
                <lc:LineSeries Values="4,6,2,5" />
            </lc:CartesianChart.Series>
            <lc:CartesianChart.AxisX>
                <lc:Axis Labels="0,1,2,3"/>
            </lc:CartesianChart.AxisX>
            <lc:CartesianChart.VisualElements>
                <lc:VisualElement X="1" Y="6"
                                   HorizontalAlignment="Center" VerticalAlignment="Top">
                    <lc:VisualElement.UIElement>
                        <TextBlock FontWeight="Bold" FontSize="16" Opacity=".6">
                           Warning!
                        </TextBlock>
                    </lc:VisualElement.UIElement>
                </lc:VisualElement>
                <lc:VisualElement X="2" Y="1" 
                                   HorizontalAlignment="Center" VerticalAlignment="Bottom">
                    <lc:VisualElement.UIElement>
                        <TextBlock FontWeight="Bold" FontSize="16" Opacity=".6">
                           All is well.
                        </TextBlock>
                    </lc:VisualElement.UIElement>
                </lc:VisualElement>
            </lc:CartesianChart.VisualElements>
        </lc:CartesianChart>
    </Grid>
</Window>
```

MainWindow.xaml.cs

```
using System.Windows;
using System.Windows.Controls;
using LiveCharts.Wpf;

namespace LiveCharts_nugetUpdater
{
    /// <summary>
    /// Interaction logic for MainWindow.xaml
    /// </summary>
    public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();
        }

        private void Window_Loaded(object sender, RoutedEventArgs e)
        {
            TheChart.VisualElements.Clear();
            VisualElement vE = new VisualElement
            {
                X = 1,
                Y = 2,
                HorizontalAlignment = HorizontalAlignment.Center,
                VerticalAlignment = VerticalAlignment.Top
            };
            vE.UIElement = new TextBlock
            {
                Text = "Add this."
            };
            TheChart.VisualElements.Add(vE);
        }
    }
}
```
#### Extra notes

I expect to be able to clear all visualelements, and then add more.
#### I am using

<table>
<tr>
    <td>Live-Charts version</td>
    <td>0.7.17</td>
</tr>
<tr>
    <td>.Net Version</td>
    <td>4.6.1</td>
</tr>
<tr>
    <td>Windows</td>
    <td>10</td>
</tr>
</table>

**Comment by FoggyFinder** [comment]
https://github.com/beto-rodriguez/Live-Charts/issues/330

**Comment by beto-rodriguez** [comment]
Thanks again for another report

Last commit should fix this issue, and will be included in version 0.8
